### PR TITLE
Moves removing the cell to screwdriver_act for APCs

### LINKED ISSUE #37628: APC covers cannot be closed while a cell is inside
Title. Caused by https://github.com/tgstation/tgstation/pull/37035

You physically cannot close an APC cover outside of clockcult as of #37035 

Crowbars and clockcult replicant alloy are the only two ways to close an APC cover. #37035 makes it so crowbarring an APC while it has a cell inside ejects the cell instead of closing the cover.

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
Fixes #37628

:cl: Naksu
fix: APCs are closed with crowbars properly again. Screwdrivers can now be used by borgs to pop out the cell instead of crowbars
/:cl:

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -457,8 +448,14 @@
 /obj/machinery/power/apc/screwdriver_act(mob/living/user, obj/item/W)
 	. = TRUE
 	if(opened)
-		if (cell)
-			to_chat(user, "<span class='warning'>Close the APC first!</span>")
+		if(cell)
```
Comment (Jalleo) [inline code/modules/power/apc.dm]: Why cant it just be set in the interact proc with removing the cell?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -457,8 +448,14 @@
...
+			user.visible_message("[user] removes \the [cell] from [src]!","<span class='notice'>You remove \the [cell].</span>")
+			var/turf/T = get_turf(user)
+			cell.forceMove(T)
+			cell.update_icon()
+			cell = null
+			charging = 0
```
[the remark this answers is not in the document]
Comment (ShizCalev) [reply code/modules/power/apc.dm]: not boolean

[the remark this answers is not in the document]
Comment (vuonojenmustaturska) [reply code/modules/power/apc.dm]: this whole code could use someone to take an axe to it

**Comment by Winterous** [comment]
Can this get fixed already? This is a stupid issue.

**Comment by Anonmare** [comment]
Can we please have this fixed already

**Comment by Anonmare** [comment]
@Cyberboss

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**